ui-meta/multinode-infra-appliance.yml

```diff
@@ -0,0 +1,183 @@
+    #   cloud.cluster
+    #     tag: An optional tag that the cluster must have.
+    # options:
+    #  min_ram: 1024
+    #  min_disk: 10
+    #  secret: true
+    #  confirm: true
+  
+# A list of parameters to be gathered from the user
+parameters:
+  - name: vxlan_vni
+    label: Set VXLAN VNI.
+    description: Please provide a VXLAN VNI. A unique value from 1 to 100,000.
+    kind: "integer"
+    required: true
+    default: null
+    immutable: true
+    options:
+      min: 1
+      max: 100000
+      
+
+  - name: ans_vlt_pwd
+    # A human-readable name for the parameter. Defaults to the name if not given.
+    label: Ansible Vault password.
+    # An optional short description for the parameter.
+    description: Please provide the Ansible Vault password to enable the deployment of the cluster.
+    kind: "string"
+    required: false
+    #default: "password_not_set"
+    immutable: false
+    # Replace the characters of the parameter with asterisks in the UI.
+    options:  
+      secret: true
+      confirm: true
+  
+  - name: openstack_deploy 
+    # A boolean flag which will be set to true if the user selects the checkbox,
+    # this will allow the user to decide whether they wish to have the
+    # infrastructure built with or without OpenStack.
+    label: Deploy Infrastructure with OpenStack services? #(True/False)
+    description: If you wish to deploy OpenStack services in the infrastructure select this option.
+    kind: "boolean"
+    default: false
+    required: false
+    immutable: true
+
+  - name: kayobe_version_branch
+    label: Kayobe GitHub Branch.
+    description: |
+      Please provide the OpenStack GitHub Branch name you wish to deploy.
+      i.e. stackhpc/yoga (for https://github.com/stackhpc/kayobe/tree/stackhpc/yoga)
+    kind: "string"
+    required: false
+    default: "stackhpc/yoga"
+    immutable: true
+
+  - name: kayobe_config_branch
+    label: Kayobe configuration GitHub Branch.
+    description: |
+      Please provide the GitHub Branch with the OpenStack configuration you wish to deploy.
+      i.e. stackhpc/yoga (for https://github.com/stackhpc/stackhpc-kayobe-config/tree/stackhpc/yoga)
+    kind: "string"
+    required: false
+    default: "stackhpc/yoga"
+    immutable: true
+ 
+
+  - name: multinode_image
+    label: Operating system image.
+    description: |
+      Please provide the OS you wish to deploy on this multinode infrastructure.
+      The image must be available in the OpenStack tenancy, two supported images are:
+      - Ubuntu-22.04-lvm
+      - rocky9-lvm
+    kind: "string"
+    required: true
+    default: "rocky9-lvm"
+    immutable: true
+
+  - name: ssh_user
+    label: Operating system SSH username.
+    description: |
+      Please select the OS username in order to connect to the infrastructure. 
+      Examples are:
+      - ubuntu (Ubuntu-22.04-lvm)
+      - cloud_user (rocky9-lvm)
+    kind: "string"
+    required: true
+    default: "cloud_user"
```

Comment:
```suggestion
      - cloud-user (Rocky9-lvm)
    kind: "string"
    required: true
    default: "cloud-user"
```

Reply:
Applied the default change and want to keep the comment about the image consistent.